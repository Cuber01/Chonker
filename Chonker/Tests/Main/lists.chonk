var list a = [[1, "string"], 2];

puts a;
puts a[1];
puts a[1][0];