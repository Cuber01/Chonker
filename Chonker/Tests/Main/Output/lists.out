list
2
string
